README.md

```diff
@@ -5,6 +5,25 @@ description: >-
 ---
 
 # Introduction
+## OpenLab
+
+OpenLab is a community-led program to test and improve support for the most popular Software Development Kits(SDKs) as well as platforms like Kubernetes, Terraform, CloudFoundry and more. The goal is to improve the usability, reliability and resiliency of tools and applications for hybrid and multi-cloud environments.
+
+## How to Join OpenLab
+
+There is plenty of room for helping the shape of future of OpenLab and ensuring things run smoothly. Welcome to join OpenLab if you're interested in our system, please follow the guide below:
+
+### To be an OpenLab member
+Join us to be a member of OpenLab firstly, then you will find out what's happening in OpenLab, what fellow open source community members are doing using OpenLab resources, also as a member you can access OpenLab resources for your testing, please follow the simple [member application](Member_Application.md) guide.
+
+### Test Request
+If you wish to use OpenLab resources, start your test request [here](./contributor/Testing_Request.md).
+
+### Development
+The main work of OpenLab CI is to provide an auto integration testing for different upstream tool chains of OpenStack, e.g. Gophercloud, Terraform. Always we have to add CI jobs for the test requests submitted to OpenLab, if you are interesting in this job, please follow [Development Guide](contributor/Development_Workflow.md).
+
+### Resource sponsoring
+The OpenLab has multiple resource pools provided by multiple cloud providers to run testing jobs. Thanks to our sponsors we also can offer the dedicated resources to users for their own tests. We'd very appreciate it if you would like to [donate](./contributor/Resource_Sponsor.md) resources to us.
 
 ## Submitting Your Publication
```

Comment:
I prefer to remove this whole section `Submitting Your Publication`, this part content have been included in `Test Request` section